docker/4.4.0-alpine/run.sh

```diff
@@ -0,0 +1,68 @@
+#/bin/bash
+
+# -------------- #
+set -x -e -u
+# -------------- #
+
+# -------------- #
+# Allow the container to be started with `--user`
+if [ "$1" = 'bookkeeper' -a "$(id -u)" = '0' ]; then
+    chown -R "$BK_USER" "${BK_DIR}" "${BK_JOURNAL_DIR}" "${BK_LEDGER_DIR}" "${BK_INDEX_DIR}"
+    exec su-exec "$BK_USER" /bin/bash "$0" "$@"
+    exit
+fi
+# -------------- #
+
+# -------------- #
+# Copy input config files in Bookkeeper configuration directory
+cp -vaf /conf/* ${BK_DIR}/conf || true
+chown -R "$BK_USER" ${BK_DIR}/conf
+
+# Bookkeeper setup
```

Comment:
for the environment variables, can we have a script:

- in the dockerfile, taking a bookkeeper template configuration and export them as the environment. for example, for a setting 'x', we can export it as 'BK_${X}'.
- in the run.sh script, looping over the template configuration and for each parameters, check if its corresponding environment variable is set or not. if it is set, replace the settings.